taskvine/src/manager/vine_current_transfers.h

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (C) 2022- The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file COPYING for details.
+*/
+
+#include "vine_worker_info.h"
+#include "uuid.h"
+
+#define VINE_FILE_SOURCE_MAX_TRANSFERS 1
+
+struct vine_transfer_pair {
```

Comment:
Put this structure definition in the C file, since it isn't used by callers of the module.

Reply:
related to previous comment, this is an argument to the hash table delete helper function used in vine_manager